automatilib/cola/backend.py

```diff
@@ -21,6 +21,7 @@ def authenticate(self, request, username: Optional[str] = None, password: Option
         """
         try:
             email = kwargs["user_response"]["email"]
+            email = email.lower()
```

Comment:
You'll have to add a case insensitive match to users because we've created some with the wrong emails now
https://stackoverflow.com/questions/64263402/django-model-objects-get-case-insensitivity

Reply:
Better docs
https://docs.djangoproject.com/en/3.1/ref/models/querysets/#iexact

Reply:
Just realised that we don't need to worry about that as we save the emails as lower case in the model: https://github.com/i-dot-ai/automatilib/blob/a422f68c7486635d329e24c6cf6f64233b6beb5b/automatilib/core/models.py#L28